modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/ExceptionUtils.cs

```diff
@@ -99,10 +99,41 @@ private static void SendToScriptDebugger(Exception e)
             }
         }
 
+        private static void InvokeUserUnhandledExceptionReporter(Exception e)
+        {
+            try
+            {
+                Godot.Logging.InvokeUnhandledExceptionReporter(e);
+            }
+            catch (Exception refailure)
+            {
+                // We try again, just in the hopes that we can attempt to report a failure in their callback system.
+                try
+                {
+                    Godot.Logging.InvokeUnhandledExceptionReporter(refailure);
+                }
+                catch (Exception)
+                {
+                    // Well, that's probably not going to work. Sorry.
+                    // Try to report something anyway.
+                    try
+                    {
+                        GD.PushError("User exception handler threw an exception, giving up");
+                    }
+                    catch (Exception)
+                    {
+                        // fine, never mind
+                    }
+                }
+            }
```

Comment:
I think try/catch is not needed. The method `LogException` already wraps the call to this method with a try/catch.